services/navigation/builtin/builtin_test.go

```diff
@@ -0,0 +1,28 @@
+// Package builtin contains the default navigation service, along with a gRPC server and client
+package builtin
+
+import (
+	"context"
+	"testing"
+
+	"github.com/edaniels/golog"
+	_ "go.viam.com/rdk/components/base/fake"
+	_ "go.viam.com/rdk/components/movementsensor/fake"
+	"go.viam.com/rdk/config"
+	robotimpl "go.viam.com/rdk/robot/impl"
+	_ "go.viam.com/rdk/services/motion/builtin"
+	"go.viam.com/rdk/services/navigation"
+	"go.viam.com/test"
+)
+
+func TestReadCfg(t *testing.T) {
```

Comment:
We *definitely* need to write some real mocked-up tests that actually capture functionality.